packages/core/test.iso/index.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
there's a root `/test/` directory that would be a great place to put the repeated parts of this file. then these `test.iso/index.js` files would import that shared code and pass their own list of components and options.